boards/arm/thingy53_nrf5340/pm_static_thingy53_nrf5340_cpuapp_ns.yml

```diff
@@ -8,16 +8,16 @@ mcuboot_pad:
   size: 0x200
 tfm_secure:
   address: 0x10000
-  size: 0xc200
+  size: 0xc000
   span: [mcuboot_pad, tfm]
 tfm_nonsecure:
-  address: 0x1c200
+  address: 0x1c000
   size: 0xd3e00
```

Comment:
`size: 0xd4000` ?

Reply:
@joerchan Cab maybe saz more about that,

app hast the same size, so that matches

Reply:
Is the `tfm_nonsecure` partition actually SPU region aligned now? Or maybe only partition start needs to be aligned?
The size of `0xd3e00` is not divisible by `0x4000`.

Reply:
`size: 0xd4000`, yeah this makes sense.
We've technically reduced the size of TF-M by 0x200, so this can be added to the application / tfm-nonsecure now.

Same for app : size below then.

Reply:
Ok changed it for both